src/pages/resources/[resourceSlug]/items/[itemId].js

```diff
@@ -86,6 +86,15 @@ const ItemPage = ({ propResource, previewMode }) => {
     }
   }, [user, itemId, resourceSlug]);
 
+  useEffect(() => {
+    if (!previewMode && item) {
+      firebase.analytics().logEvent('select_content', {
+        content_type: 'resource_content_item',
+        item_id: `${resource.slug}/${itemId}`,
+      });
+    }
+  }, [item, itemId, resourceSlug]);
```

Comment:
Because you've got item & itemId here this could fire twice, on a route change

Not sure thought, needs checking

Reply:
Will look into this – it did _seem_ like some events were being fired twice, so let me double check.

Reply:
Looks like it only fires once. So all good.